bin/radical-pilot-bridge

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+
+__copyright__ = "Copyright 2014-2019, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+
+import sys
+import time
+
+import setproctitle  as spt
+import radical.utils as ru
+
+
+# ------------------------------------------------------------------------------
+#
+def main(cfg):
+    '''
+    This thin wrapper starts a ZMQ bridge.  It expects a single argument:
+    a config to use for the bridge's configuration.  The config must contain:
+
+      - uid : UID of bridge instance (unique to the hosting session)
+      - name: name of the bridge
+      - kind: type of bridge (`pubsub` or `queue`)
+
+    If the config contains a `heartbeat` section, that section must be formatted
+    as follows:
+
+        {
+          'from'    : 'uid',
+          'pub'     : 'addr_pub',
+          'sub'     : 'addr_sub',
```

Comment:
We use `pub`/`sub` for the heartbeat. Does this mean that this section is only for the `pubsub` type of bridge?

Reply:
Yes, heartbeats go over a pubsub channel.

Reply:
Can we say this explicitly in the docstring?

Reply:
I may have misunderstood you?  Not sure.  That config section:
```
{
          'from'    : 'uid',
          'pub'     : 'addr_pub',
          'sub'     : 'addr_sub',
...
```
is to connect the bridge to our heartbeat monitor.  The bridge itself can be for pubsub channels or queues or whatever.  The heartbeats do not run over a bridge (chicken & egg, you see :-)).

Did that answer make more sense?  I might still be off the mark...

Reply:
Thanks, this helps. Suggested small change in documentation: ```RCT heartbeats do not run over a bridge so we may want to connect a bridge to an heartbeat monitor. If the config contains a `heartbeat` section, that section must be formatted as follows:``` Would this work? The origin of this comment was that without knowing that implementation detail, I was confused about the relationship between heartbeat and bridge.

Reply:
Great, I fixed the doc.